manifests/cf/cloud_controller.yml

```diff
@@ -13,6 +13,9 @@ instance_groups:
         agent:
           services:
             cloud_controller_ng: {}
+            policy-server-internal:
+              name: policy-server
+
```

Comment:
Might want to include a comment here as to why the name and the key differ (iirc, it's because of the health check path)